src/pages/view-test-result/cat-cpc/components/vehicle-details-card/vehicle-details-card.html

```diff
@@ -5,7 +5,7 @@ <h4>Vehicle details</h4>
   <ion-card-content>
     <ion-grid>
       <data-row [label]="'Vehicle registration number'" [value]="vehicleDetails.registrationNumber"></data-row>
-      <data-row [label]="'Vehicle details'" [value]="vehicleDetails.configuration" [shouldHaveSeperator]="false"></data-row>
+      <data-row *ngIf="vehicleDetails.configuration" [label]="'Vehicle details'" [value]="vehicleDetails.configuration" [shouldHaveSeperator]="false"></data-row>
```

Comment:
How comes this extra condition was necessary?

Reply:
When showing DCPC test it will display the vehicle row with no data

![Simulator Screen Shot - iPad Pro (10 5-inch) - 2020-11-18 at 16 45 53](https://user-images.githubusercontent.com/17671699/99560303-a3e23200-29bd-11eb-9eae-23090b5cdceb.png)